package-res/resources/web/pentaho/visual/ccc/_axes/AbstractAxis.js

```diff
@@ -21,7 +21,7 @@ define([
   "use strict";
 
   return Base.extend({
-    constructor: function(chart, axisId, gems) {
+    constructor: function(chart, axisId, mais) {
       this.chart = chart;
```

Comment:
`mais` means `mappingAttributeInfos` and I have abbreviated it, in some places, as `mappingAttrInfos`. Even this abbreviation I find long to a point that it prejudices readability, so I many times used the shorted `mais`.

Like with any abbreviation, initially it is harder to read, but then it speeds up reading and comprehension.

If you look at the code from the beginning (i.e., at least, starting from the `abstract/View` file) you will find many annotations and comments that clarify the abbreviation.